pkg/api/types.go

```diff
@@ -1,8 +1,9 @@
 package api
 
 import (
-	docker "github.com/fsouza/go-dockerclient"
 	"time"
+
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
@enoodle goimports reordered this imports automatically for me (vim-go). what merge problems you have in mind?